handlers/facebook/facebook.go

```diff
@@ -37,6 +37,14 @@ var (
 		369239343222814: "👍", // medium
 		369239383222810: "👍", // big
 	}
+
+	tagByTopic = map[string]string{
+		"":         "ACCOUNT_UPDATE",
```

Comment:
need a default for a msg with no topic

Reply:
or do we keep sending with `NON_PROMOTIONAL_SUBSCRIPTION` and no tag for now?

Reply:
I think that's right yes, if we don't have a reply then we continue to use the old one. (it may fail but at least we tried)

Reply:
Updated so it's

 1. If it's a response, use message type = `RESPONSE`, no tag
 2. If there's a topic, use message type = `MESSAGE_TAG`, with tag
 3. Use message type = `NON_PROMOTIONAL_SUBSCRIPTION`, no tag

We can revisit nearer to Jan 15th